### ISSUE TITLE
Вынести автообновление в отдельную галочку

### ISSUE THREAD

**Issue by anvme** [ziemenz/Kinopoisk.bundle#16]
Сейчас автообновление отключается только через список выбора - none, stable, beta.
Канал обновления (none - отключение автообновления) [список]
Предлагаю переделать так
Канал обновления [список] v Разрешить автообновление

**Comment by Anisotropic** [comment]
Да, было бы неплохо. А то у меня похоже автообновление нормально не отрабатывает - стирает всё содержимое kinopoisk.bundle, а новое не ставит
